### PR TITLE
Rename /success to /manage and fix link in calendar event

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

- Renames `/success` link to `/manage`
- Fixes 'Need to reschedule or cancel?' link in calendar event (and scrolls down to bottom of screen)

## How should this be tested?

- Book a meeting 
- Test new  'Need to reschedule or cancel?' link in calendar event 
- Test cancel link in confirmation email 
- Test old `/success` links from emails and calendar events

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by PeerRich** [comment]
will old links still redirect?

File: apps/web/pages/manage.tsx [hunk 1]
```diff
@@ -0,0 +1,980 @@
...
+  const router = useRouter();
+
+  const {
+    allRemainingBookings,
+    isSuccessBookingPage,
+    cancel: isCancellationMode,
+    changes,
+  } = querySchema.parse(router.query);
+
+  if ((isCancellationMode || changes) && typeof window !== "undefined") {
```
Comment (CarinaWolli) [inline apps/web/pages/manage.tsx:166]: Code copied from the success.tsx file. The only changes in this file are the new URL parameter 'changes' which is set to true for the 'Need to reschedule or cancel?' Link in the calendar event which makes it scroll down automatically in smaller screens

File: apps/web/pages/success.tsx [hunk 2]
```diff
@@ -1,978 +1,18 @@
...
   return {
-    props: {
-      hideBranding: eventType.team ? eventType.team.hideBranding : eventType.users[0].hideBranding,
-      profile,
-      eventType,
-      recurringBookings: recurringBookings ? recurringBookings.map((obj) => obj.startTime.toString()) : null,
-      trpcState: ssr.dehydrate(),
-      dynamicEventName: bookingInfo?.eventType?.eventName || "",
-      bookingInfo,
+    redirect: {
```
Comment (CarinaWolli) [inline apps/web/pages/success.tsx:13]: @PeerRich here is the redirect from the old /success link to /manage.

Comment (PeerRich) [reply apps/web/pages/success.tsx:13]: ah nice

Comment (CarinaWolli) [reply apps/web/pages/success.tsx]: Good point. I deleted the success.tsx file and use nextjs rewrites instead

File: apps/web/next.config.js [hunk 3]
```diff
@@ -149,6 +149,10 @@ const nextConfig = {
         source: "/router",
         destination: "/apps/routing-forms/router",
       },
+      {
```
Comment (CarinaWolli) [inline apps/web/next.config.js:152]: Rewrites the old `/success` link with all URL paramaters with `/manage`

**Review by JeroenReumkens — APPROVED** [review]
Looks good to me! 🙏

**Review by zomars — CHANGES_REQUESTED** [review]
Let's keep the booking permalink at `/booking/[uid]` 🙏🏽

**Comment by CarinaWolli** [comment]
@zomars changed the link to `/booking/[uid]`, it is ready for review again

File: apps/web/pages/cancel/[uid].tsx [hunk 4]
```diff
@@ -18,7 +18,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   return {
     redirect: {
       permanent: false,
-      destination: `/success?uid=${uid}&allRemainingBookings=${allRemainingBookings}&cancel=true`,
+      destination: `/booking/${uid}?allRemainingBookings=${allRemainingBookings}&cancel=true`,
```
Comment (zomars) [inline apps/web/pages/cancel/[uid].tsx:21]: @CarinaWolli can we add a next rewrite for this page as well?

**Review by zomars — APPROVED** [review]
Ship it

**Pull request merged**